sharding/mock/enableEpochsHandlerStub.go

```diff
@@ -0,0 +1,394 @@
+package mock
+
+type EnableEpochsHandlerStub struct {
+	WaitingListFixEnableEpochField uint32
+}
+
+// BlockGasAndFeesReCheckEnableEpoch returns 0
+func (stub *EnableEpochsHandlerStub) BlockGasAndFeesReCheckEnableEpoch() uint32 {
```

Comment:
this is not a stub, but rather a mock, as you always return 0. also, don't we have this stub somewhere else?

Reply:
renamed it to `enableEpochsHandlerStub`..indeed we do have the stub in `testscommon`, but using it into `nodesCoordinator` package was leading to cyclic imports.

I decided that creating a new mock, with the needed functionality for this package was an easier change than moving all the tests from package `nodesCoordinator` to a new `nodesCoordinator_test` package, as they directly use a lot of unexported methods